src/components/Dashboard/DashboardTile/DashboardTile.js

```diff
@@ -510,10 +510,13 @@ export class DashboardTile extends React.Component {
 
     return Promise.all(promises)
       .then((queryResponses) => {
+        // Helper to normalize query response
+        const normalizeResponse = (resp) => (Array.isArray(resp) ? { data: { rows: resp } } : resp ?? {})
```

Comment:
Why would the queryResponse be an array? Should we not find the source of that bug instead of converting it to an object here?